### ISSUE TITLE
kernel_module doesn't work in CentOS 7.2

### ISSUE THREAD

**Issue by aladmit** [inspec/inspec#1507]
### Description

Kernel_module doesn't work in CentOS 7.2

### InSpec and Platform Version

InSpec 1.11.0
ChefDK 1.2.22
My OS: macOS Sierra 10.12.3
Server's OS: CentOS 7.2

### Replication Case

```
describe kernel_module('ip_tables') do
  it { should be_loaded }
end
```

### Possible Solutions

`command('lsmod').stdout` return empty string, but `command('/sbin/lsmod')` return normal output. So, I think we should remove condition about [`inspec.os[:release]`](https://github.com/chef/inspec/blob/master/lib/resources/kernel_module.rb#L27) from kernel_module.

### Stacktrace

```
inspec> describe kernel_module('ip_tables') do
inspec>   it { should be_loaded }
inspec> end

Profile: inspec-shell
Version: unknown

  Kernel Module
     ∅  ip_tables should be loaded
     expected `Kernel Module ip_tables.loaded?` to return true, got false
```

**Comment by aladmit** [comment]
Yeah, my solution is work, I'll send PR.

**Comment by adamleff** [comment]
Fixed in #1513, and there's a pending PR #1798 that cleans this up even further.
